src/collective/behavior/banner/browser/viewlets.py

```diff
@@ -76,7 +77,8 @@ def banner(self, obj):
         if obj.banner_title:
             banner['banner_title'] = obj.banner_title
         if obj.banner_description:
-            banner['banner_description'] = obj.banner_description
+            crop = Plone(self.context, self.request).cropText
```

Comment:
The only reason is that I copied it from the tests in CMFPlone, which I understand cannot use plone.api.